core/lib/dal/src/blocks_dal.rs

```diff
@@ -986,14 +986,21 @@ impl BlocksDal<'_, '_> {
         sqlx::query!(
             r#"
             INSERT INTO
```

Comment:
Another example of lenient formatting: the formatter would accept here

```rust
        sqlx::query!(
            r#"
            INSERT INTO commitments (
                l1_batch_number,
                events_queue_commitment,
                bootloader_initial_content_commitment
            )
            VALUES ($1, $2, $3)
            ON CONFLICT (l1_batch_number) DO NOTHING
            "#,
            _
        )
```

or even a monstrosity like

```rust
        sqlx::query!(
            r#"
            INSERT
            INTO commitments (
                l1_batch_number, events_queue_commitment,
                bootloader_initial_content_commitment
            )
            VALUES (
                $1, $2, $3
            )
            ON CONFLICT
            (l1_batch_number)
            DO NOTHING
            "#,
            _
        )
```

Reply:
But, i'd say the original option was even worse, no?